controllers/v1/users.js

```diff
@@ -149,11 +149,13 @@ module.exports = class Users {
 		return new Promise(async (resolve, reject) => {
 			try {
 				let isAPrivateProgram = UTILS.convertStringToBoolean(req.query.isAPrivateProgram)
+				req.query.isProjectsCount = UTILS.convertStringToBoolean(req.query.isProjectsCount)
```

Comment:
@praveenKDass we can call the query getProjectsCount

Reply:
Changed @VISHNUDAS-tunerlabs